packages/vscode-extension/package.json

```diff
@@ -83,7 +85,8 @@
   "activationEvents": [
     "workspaceContains:**/.theme-check.yml"
   ],
-  "main": "./dist/extension.js",
+  "main": "./dist/node/extension.js",
+  "browser": "./dist/browser/extension.js",
```

Comment:
Major difference is here: We have a different build for web that uses `theme-language-server-browser`